### DISCUSSION TITLE
How to glob import fonts

### DISCUSSION THREAD

**Discussion by LIMPIX31** [vanilla-extract-css/vanilla-extract#1178] [Q&A] [answered]
I want to assign fonts to my theme. I realize that the following won't work, since `vanilla-extract` is run at build time using esbuild.
```ts
export const fonts = import.meta.globEager('./*.woff2', { import: 'default' }) 
```
Is there any way to import fonts (or rather paths to them) while in the `vanilla-extract` environment?

**Comment by askoufis** [comment, marked as answer]
While you are restricted to the `esbuild` environment, you can do basically anything you want at build-time, including using [a glob library](https://www.npmjs.com/package/glob) to get file paths.
